packages/map/src/popup/index.jsx

```diff
@@ -45,6 +67,7 @@ const Popup = ({
 Popup.defaultProps = {
   offsetX: 0,
   offsetY: 0,
+  closeButton: true,
```

Comment:
`closable`?

Reply:
closeButton only shows the close button itself, the popup is always closable